playwright/tests/e2e/features/transaction-overview.feature

```diff
@@ -32,30 +32,30 @@ Feature: Check Transaction Cardano Explorer
 
   Scenario: Transaction detail page through info widget 
     Given the user is in the Transactions page for go to transaction detail from detail button in widget
-    Given the user open the info widget of one transaction record in the transactions table of transaction page
+    And the user open the info widget of one transaction record in the transactions table of transaction page
```

Comment:
Steps in line 35, 41, 47, 53 and 59 are the same step, basically open the info widget of the last transaction we can refactor this step name and re use it in the other test so we don't duplicate code.